packages/icons/src/chaos-experiment-weight.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 19 19" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
please check https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

we should use `currentColor` for fill instead of a hardcoded hex if this is a line-icon and not a logo

Reply:
Updated `fill` property to use `currentColor` except for chaos-logo since it's a logo.